docs/general/Introduction/introduction.md

```diff
@@ -6,6 +6,8 @@ sidebar_position: 1
 
 Movement Labs is building a network of Move-based blockchains. Our flagship products are the Movement Network and the M1 shared sequencer. 
 
+> ⚠️ This documentation provides a high level intro to the vision. The [white paper](https://www.movementnetwork.xyz/whitepaper/movement-whitepaper_en.pdf) goes into more details on some of the presented solutions.
+
 Movement is a community-first blockchain providing the highest possible TPS through Move, instant finality, native day-zero access to mass liquidity, and modular customizations.
 
 It will support Aptos Move, Sui Move, and also our embedded EVM interpreter MEVM—empowering both Sui, Aptos, and EVM users to use the L2.
```

Comment:
Maybe this could be worded as a little more ambiguous for Sui Move and MEVM
```suggestion
It will support Aptos Move, Sui Move, and also our embedded EVM interpreter MEVM—empowering both Sui, Aptos, and EVM users to use the L2.
```

Reply:
i dont think this belongs here. i removed it. also our flagship project is not M1 but Movement Network. fixed this.